app/src/index.html

```diff
@@ -11,8 +11,8 @@
   <!-- Facebook OG tags -->
   <meta property="og:title" content="Academix | Sustainable Education Foundation"/>
   <meta property="og:type" content="website"/>
-  <meta property="og:url " content="https://sefglobal.org/academix/"/>
-  <meta property="og:image" content="https://sefglobal.org/academix/img/academix-logo.png"/>
+  <meta property="og:url" content="https://sefglobal.org/academix/"/>
+  <meta property="og:image" content="https://sefglobal.org/academix/img/academix-og-image.png"/>
```

Comment:
This should be https://sefglobal.org/academix/public/academix-og-image.png, right?

Reply:
Thank for the guidance in this!